packages/pymedphys_dicom/src/pymedphys_dicom/dicom/_level2/anonymise.py

```diff
@@ -247,7 +248,7 @@ def anonymise_file(
         This ensures that the filename contains no identifying
         information. If set to `False`, `anonymise_file()` simply
         appends "_Anonymised" to the original DICOM filename. Defaults
-        to True.
+        to `True`.
```

Comment:
> How often do DICOM files have identifying information in their file name? Part of me thinks this is overstepping the scope of the function. But it's not something I'll hold back a merge over.

Varian's preferred format is:
`"<Modality>.<PatientID>.<optional extras - like plan name>.dcm"`

So all the time, if you count `PatientID`...

> Then again, that file name format is quite neat. You've convinced me :)... And you did without even speaking :).

Hahaha, I just mimicked RaySearch and Philips' (and apparently everyone except Varian's) convention :)

> For a single file (as opposed to a directory) the API for these sorts of things usually are input file, output file.

Sure. Where in particular do you want that reflected though? Just in the CLI functions? Because I really don't like variable names like `input_file` in the backend - it's not clear that it's a filepath, file handler/pointer, etc. It's fine in the CLI, since the user kinda *has to* supply a filepath...

> Seeing that written, wouldn't it be really nice of we did have a top level, plain, Dicom object that was able to appropriately mixin the methods of various Dicom types depending on what the type was of the DICOM file stored within the file

It certainly would :)